src/component/Navbar.js

```diff
@@ -0,0 +1,18 @@
+import React from "react"
+import { Link } from 'react-router-dom'
+
+// This Navbar allows users to navigate from the List view to the Add Item view
+// It is used in the FetchItems.js component and the AddItem.js component
+
+function Navbar() {
+    return (
+        <React.Fragment>
```

Comment:
You only need to use <React.Fragment> when you would otherwise have multiple nodes at the top level of the component. In this case, you have a <div> that can act as that single top-level component, so you can safely remove the Fragment tag.